snippets/ai/src/ai.ts

```diff
@@ -0,0 +1,176 @@
+import type { AiProvider } from './providers/ai-provider.js';
+import type { Config, ConfigSchema } from './config.js';
+import type { CliContext } from './helpers.js';
+import type { Models } from './providers/generic/ai-sdk-provider.js';
+
+module.exports = (async (globalThis: CliContext) => {
+  const _localRequire = require('module').createRequire(__filename);
+  const localRequire = <T>(module: string): T => _localRequire(module);
+
+  const { aiCommand } = localRequire<typeof import('./decorators.js')>('./decorators.js');
+  const { EmptyAiProvider } = localRequire<typeof import('./providers/ai-provider.js')>('./providers/ai-provider.js');
+  const { getDocsAiProvider } = localRequire<typeof import('./providers/docs/docs-ai-provider.js')>('./providers/docs/docs-ai-provider.js');
+  const { getAiSdkProvider, models } = localRequire<typeof import('./providers/generic/ai-sdk-provider.js')>('./providers/generic/ai-sdk-provider.js');
+  const { Config } = localRequire<typeof import('./config.js')>('./config.js');
+  const { wrapAllFunctions } = localRequire<typeof import('./helpers.js')>('./helpers.js');
+  const chalk = localRequire<typeof import('chalk')>('chalk');
```

Comment:
You shouldn't need `localRequire` in any file except the main entry point (index.js)